Utilities/GenerateDocs/README

```diff
@@ -15,9 +15,17 @@ the SwigDocUpdate.sh script can be run to convert the SimpleITK
 Doxygen XML into Swig .i files so that when SimpleITK is built, inline
 documenation for some wrapped languages will be available.
 
+Building the documentation target from the SuperBuild framework requires:
+        The correct cmake flags: -DBUILD_DOXYGEN:BOOL=ON -DBUILD_DOCUMENTS:BOOL=ON
```

Comment:
The "-DBUILD_DOCUMENTS:BOOL=ON"flag  is not needed and should be removed.

There is also a super build target "doc" could be run.